src/pages/Home/Home.jsx

```diff
@@ -1,15 +1,24 @@
-import React from 'react';
+import React, {useEffect} from 'react';
+import ErrorBoundary from 'pages/shared/ErrorBoundary/ErrorBoundary';
 import Navigation from 'pages/Home/components/Navigation/Navigation'
 import MovieList from 'pages/Home/components/MovieList/MovieList';
-import ErrorBoundary from "pages/shared/ErrorBoundary/ErrorBoundary";
+import Search from "pages/Home/components/Search/Search";
 import 'pages/Home/Home.scss';
 
-function Home() {
+const Home = (props) => {
+    const {handleSearchIcon} = props;
+
+    useEffect(() => {
+            handleSearchIcon();
+        },
+        [])
+
```

Comment:
Hi Natalia,

I assume you can avoid using useEffect in this component, just use as provider for MovieList <MovieList handleSearchIcon={props.handleSearchIcon}/>

Reply:
Hi Natalia,
Totally agree with you, deleted this unnecessary `useEffect`.